client/my-sites/people/team-list/team.jsx

```diff
@@ -44,6 +50,20 @@ class Team extends React.Component {
 			} );
 		let headerText = this.props.translate( 'Team', { context: 'A navigation label.' } );
 		let people;
+		let headerText;
```

Comment:
Looks like this is already declared two lines above. I think we can remove it here?

Reply:
Good catch, I actually removed the other one :)